src/ui_components/login_form/login_form.js

```diff
@@ -5,8 +5,8 @@ import SubmitButton from '../submit_button/submit_button';
 const LoginForm = (props) => {
   return (
     <form onSubmit={props.handleLogin}>
-      <input placeholder="Email" type="email" name="emailAddress"/><br/>
-      <input placeholder="Password" type="password" name="password"/><br/><br/>
+      <input id="Email" placeholder="Email" type="email" name="emailAddress"/><br/>
```

Comment:
Why did you add an #id here? Are you referencing it somewhere?

Reply:
#id resolved and it's not there anymore